modules/web-test/variables.tf

```diff
@@ -0,0 +1,115 @@
+variable "resource_group_name" {
+  description = "The name of the resource group to create the resources in."
+  type        = string
+}
+
+variable "location" {
+  description = "The location to create the resources in."
+  type        = string
+}
+
+variable "component_id" {
+  description = "The Application Insights component ID."
+  type        = string
+}
+
+variable "name" {
+  description = "The name of this Web Test."
+  type        = string
+}
+
+variable "configuration" {
+  description = "An XML configuration specification for this Web Test."
+  type        = string
+  default     = null
+}
+
+variable "kind" {
+  description = "The kind of web test that this Web Test watches."
+  type        = string
+  default     = "standard"
+
+  validation {
+    condition     = contains(["standard", "ping", "multistep"], var.kind)
+    error_message = "Web Test kind must be \"standard\", \"ping\" or \"multistep\"."
+  }
+}
+
+variable "description" {
+  description = "The purpose/user defined description of this Web Test."
+  type        = string
+  default     = null
+}
+
+variable "enabled" {
+  description = "Is this Web Test enabled?"
+  type        = bool
+  default     = false
+}
+
+variable "frequency" {
+  description = "Interval in seconds between test runs for this Web Test."
+  type        = number
+  default     = 300
+}
+
+variable "retry_enabled" {
+  description = "Is retry on Web Test failure enabled?"
+  type        = bool
+  default     = false
+}
+
+variable "timeout" {
+  description = "Seconds until this Web Test will timeout and fail."
+  type        = number
+  default     = 30
+}
+
+variable "geo_locations" {
+  description = "A list of physical locations to run this Web Test."
+  type        = list(string)
+  default     = ["emea-gb-db3-azr"] # North Europe
+}
+
+variable "request" {
+  description = "The request generated by this Standard Web Test."
+
+  type = object({
+    url                              = string
+    body                             = optional(string, null)
+    follow_redirects_enabled         = optional(bool, true)
+    http_verb                        = optional(string, "GET")
```

Comment:
I see in docs that there is no default value. Have you checked that it's set to "GET" by default if no value is provided. If yes, then I agree that we can set default value to "GET".

Reply:
This comment was meant for the `http_verb` property only.

Reply:
The default values set for variable arguments, that are without default values in the docs, were set in the early phase based on insubsea configuration. I just forgot to set them back 😁 But I will check what default value is set when no value is provided 👍🏼

Reply:
Can confirm that `http_verb` has the default value `"GET"`.